handlers/handlers.go

```diff
@@ -27,3 +27,12 @@ func GeographyStart(cfg config.Config, rc RenderClient) http.HandlerFunc {
 		rc.BuildPage(w, model, "geography-start")
 	}
 }
+
+// GeographyStart Handler
```

Comment:
Comments for public functions start with the function name
```suggestion
// GetArea Handler
```

Reply:
Corrected the copy paste error